terragrunt-config/environments/qa/eks_autoscaler/terragrunt.hcl

```diff
@@ -0,0 +1,13 @@
+/*BC: the following functions help to access parent folders, to get to the main terraform files*/
+include {
+  path = find_in_parent_folders()
+}
+ /* BC: In this line you can reference either the direct folder of your terraform files, or github submodules. */
+terraform {
+  source = "../../src/3-eks-cluster-autoscaler"
+}
+/*BC: These variables serve as input parameters for a development/qa/production environment, for more details you can read the readme with the different values.*/
+inputs = {
+  environment = "qa"
+  aws_region = "us-east-2"
```

Comment:
What about all of these inputs? https://github.com/ManagedKube/kubernetes-ops/blob/main/terraform-environments/aws/dev/25-eks-cluster-autoscaler/main.tf#L72-L75

Reply:
in the original file,   this two inputs was filling with the same region and environment name, 
     aws_region                  = local.aws_region
     cluster_name                = local.environment_name
however I can make a better one to receive them.


-------------------------------------------------------------------------------------------
For this two inputs, This inputs is in Eks input , this is a dependence of that, 

 eks_cluster_id              = data.terraform_remote_state.eks.outputs.cluster_id
  eks_cluster_oidc_issuer_url = data.terraform_remote_state.eks.outputs.cluster_oidc_issuer_url

do you need to merge "eks and eks_autoscalar" solution?

Reply:
You can use https://terragrunt.gruntwork.io/docs/reference/config-blocks-and-attributes/#dependency to get the other outputs.